services/web/server/tests/unit/with_dbs/03/workspaces/test_workspaces.py

```diff
@@ -0,0 +1,116 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-statements
+from http import HTTPStatus
+
+import pytest
+from aiohttp.test_utils import TestClient
+from models_library.api_schemas_webserver.workspaces import WorkspaceGet
+from pytest_simcore.helpers.assert_checks import assert_status
+from pytest_simcore.helpers.webserver_login import UserInfoDict
+from servicelib.aiohttp import status
+from simcore_service_webserver.db.models import UserRole
+from simcore_service_webserver.projects.models import ProjectDict
+
+# NOTE: MD: test access rights
+
+
+@pytest.mark.parametrize("user_role,expected", [(UserRole.USER, status.HTTP_200_OK)])
```

Comment:
you do not test Guest, admin, and whatever. Why? there should be at least 1 test doing that.